TheJumpCloud/jumpcloud-ADMU
# Sa 3195 url cdn update

## Issues
* [SA-3195](https://jumpcloud.atlassian.net/browse/SA-3195) - Update S3 Agent Install url with CDN url

## What does this solve?

Update JumpCloud S3 bucket Agent installation exe with the new url link

## Is there anything particularly tricky?

## How should this be tested?

- On a system without JumpCloud installed
- Perform a migration on a user while checking the box in the GUI to install the JumpCloud agent. 
- The agent should install and complete the migration successfully

## Screenshots

[SA-3195]: https://jumpcloud.atlassian.net/browse/SA-3195?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
